snapshots/ilostat/2023-09-19/unemployment.csv.dvc

```diff
@@ -0,0 +1,33 @@
+# Learn more at:
+# http://docs.owid.io/projects/etl/en/latest/architecture/metadata/reference/origin/
+meta:
+  origin:
+    # Data product / Snapshot
+    title: Education and Mismatch Indicators (EMI) - Unemployment
```

Comment:
Maybe it would be better to define `title=Education and Mismatch Indicators (EMI)` (the data product), and `title_snapshot=Education and Mismatch Indicators (EMI) - Unemployment`. Would that make sense? (I haven't checked the original data, this is just my guess).